scaffoldmaker/meshtypes/meshtype_3d_ellipticcylinder1.py

```diff
@@ -0,0 +1,476 @@
+"""
+Generates a 3-D elliptic cylinder model.
+"""
+
+from __future__ import division
+
+import math
+
+from scaffoldmaker.meshtypes.scaffold_base import Scaffold_base
+from scaffoldmaker.utils.meshrefinement import MeshRefinement
+from scaffoldmaker.utils.eftfactory_bicubichermitelinear import eftfactory_bicubichermitelinear
+from scaffoldmaker.utils.eftfactory_tricubichermite import eftfactory_tricubichermite
+from scaffoldmaker.utils.zinc_utils import *
+from scaffoldmaker.utils.geometry import *
+from scaffoldmaker.utils.interpolation import *
+from scaffoldmaker.utils.vector import *
```

Comment:
The function names in vector are so generic I prefer this to be imported with:
import scaffoldmaker.utils.vector as vector
& all functions called with vector.magnitude etc.